tensorflow_datasets/__init__.py

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Lint as: python3
+
```

Comment:
Sorry about this, but could you remove the extra line here, as the example I wrote in https://github.com/tensorflow/datasets/pull/1553#issuecomment-593559304 ?

Looks good otherwise.